plugins/woocommerce/src/Gateways/PayPal/AddressRequirements.php

```diff
@@ -0,0 +1,348 @@
+<?php
+
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\Gateways\PayPal;
+
+/**
+ * Class AddressRequirements
+ *
+ * This helper class checks country-specific address requirements.
+ * This was built based on https://developer.paypal.com/api/rest/reference/orders/v2/country-address-requirements/
+ */
+class AddressRequirements {
+	/**
+	 * Countries that require a city in the address.
+	 *
+	 * @var array
+	 */
+	private const COUNTRIES_REQUIRING_CITY = array(
+		'AF',
```

Comment:
Nit: For both this list and for `COUNTRIES_REQUIRING_POSTAL_CODE`, having the values not be in alphabetical order makes it harder to skim the list to see if a specific country code is in the list. While it looks like this mirrors the sequence in the source page, it feels like ordering this now will make it easier to debug and maintain the list in the long term, especially if PayPal decide to change how they present this information.